saas_demo/os.py

```diff
@@ -120,6 +125,84 @@ def update_config(section, key, value):
         config_parser.write(configfile)
 
 
+def _fileopen(path, mode, basedir, pathinfo):
+    name = os.path.normpath(os.path.normcase(os.path.join(basedir, path)))
+    if os.path.isfile(name):
+        if 'b' in mode:
+            fo = open(name, mode)
+        else:
+            fo = io.open(name, mode, encoding='utf-8')
+        if pathinfo:
+            return fo, name
+        return fo
+
+
+def file_open(name, mode="r", pathinfo=False):
+    if os.path.isabs(name):
+        base, name = os.path.split(name)
+        return _fileopen(name, mode=mode, basedir=base, pathinfo=pathinfo)
+
+
+def adapt_version(version):
+    serie = release.major_version
+    if version == serie or not version.startswith(serie + '.'):
+        version = '%s.%s' % (serie, version)
+    return version
+
+
+def load_information_from_description_file(module, mod_path=None):
+    """
+    :param module: The name of the module (sale, purchase, ...)
+    :param mod_path: Physical path of module, if not providedThe name of the module (sale, purchase, ...)
+    """
+    if not mod_path:
+        mod_path = get_module_path(module, downloaded=True)
```

Comment:
Does it work? You use built-in method which search in addons-path only

Reply:
In our case, this condition is not met, I can remove it and remove the default value for the mod_path

Reply:
what if I check the `odoo.tools.config parameter ['test_enable']` instead?

Reply:
Yes, please